src/DataCollection.Shared/ViewModels/AppInfoViewModel.cs

```diff
@@ -0,0 +1,153 @@
+        /// </summary>
+        public bool ShowRuntimeVersion { get => Settings.Default.ShowRuntimeVersion ?? false; }
+
+        /// <summary>
+        /// Gets a value indicating if the version of the app should be shown in the UI.
+        /// </summary>
+        public bool ShowAppVersion { get => Settings.Default.ShowAppVersion ?? false; }
+
+        /// <summary>
+        /// Gets a value indicating whether a link to license info should be shown in the UI.
+        /// </summary>
+        public bool ShowLicenseInfo { get => Settings.Default.ShowLicenseInfo ?? false; }
+
+        /// <summary>
+        /// Gets a value indicating whether ShowAppInfo.
+        /// </summary>
+        public bool ShowAppInfo { get => ShowAppVersion || ShowRuntimeVersion || ShowLicenseInfo; }
+
+        /// <summary>
+        /// Gets a formatted string including the app's name, platform, and version.
+        /// </summary>
+        public string AppVersion
+        {
+            get
+            {
+                if (_appVersion == null)
+                {
+
+                    var appName = Resources.GetString("TitleBar_Name_App");
+                    var platform = "";
+#if NET_CORE
+                    platform = "WPF (.NET Core)";
+#elif WPF
+                    platform = "WPF (.NET Framework)";
+#elif NETFX_CORE
+                    platform = "UWP";
+#endif
+
+                    var version = Assembly.GetExecutingAssembly().GetName().Version.ToString();
+
+                    _appVersion = $"{appName}\n{platform}\n{version}";
+                }
+                return _appVersion;
+            }
+        }
+
+        /// <summary>
+        /// Gets the version of Runtime used by the app.
+        /// </summary>
+        public string RuntimeVersion
+        {
+            get
+            {
+                if (_runtimeVersion == null)
+                {
+                    #if NETFX_CORE
+                    var version = FileVersionInfo.GetVersionInfo(System.IO.Path.Combine(Windows.ApplicationModel.Package.Current.Installed­Location.Path, "RuntimeCoreNet.dll"));
+                    #else
+                    var runtimeTypeInfo = typeof(ArcGISRuntimeEnvironment).GetTypeInfo();
+                    var version = FileVersionInfo.GetVersionInfo(runtimeTypeInfo.Assembly.Location);
+                    #endif
+                    var buildparts = version.FileVersion.Split('.');
+                    var build = buildparts[buildparts.Length - 1];
+                    var productVersion = version.ProductVersion;
+
+                    if (productVersion.EndsWith(build))
+                    {
+                        productVersion = productVersion.Substring(0, productVersion.Length - (build.Length + 1));
+                    }
+                    _runtimeVersion = $"ArcGIS Runtime {productVersion} ({build})";
+                }
+                return _runtimeVersion;
+            }
+        }
```

Comment:
Ditto from above- just a thought here, feel free to discard based on my misunderstanding of the role of a view model. I would consider chopping these values up into two separate variables, allowing the view to determine how they are formatted rather than assuming a formatted string with static prefix "ArcGIS Runtime [..] [..]" is wanted.